binary/reference_objects.py

```diff
@@ -48,9 +48,6 @@ def __init__(self, most_sig_bits, least_sig_bits):
     },
     'BitmapIndexOptions': {
         'uniqueKeyTransformation': 1
-    },
-    'ProtocolType': {
```

Comment:
We should add `    'EndpointQualifier': {
        'type': 2
    }` since right now the binary encoder is writing 25 (default value for int) to the binary files but there is no enum for the id of 25.